src/stores/store/eth1/local/parser.go

```diff
@@ -1,12 +1,12 @@
 package local
 
 import (
-	"github.com/consensys/quorum-key-manager/src/stores/store/database/models"
 	"github.com/consensys/quorum-key-manager/src/stores/store/entities"
+	"github.com/consensys/quorum-key-manager/src/stores/store/models"
 	"github.com/ethereum/go-ethereum/crypto"
 )
 
-func parseKey(key *entities.Key, attr *entities.Attributes) *models.ETH1Account {
+func parseKey(key *models.Key, attr *entities.Attributes) *models.ETH1Account {
```

Comment:
Trying to apply a more strict idea of how Hex Arch work, models should never live outside of our /database, our entities are the respentation of our application logic and those are the elements we should move around. Therefore I would suggest to encapsulate all this under /database aiming to prevent same issues we go in Orchestrate where we got recurrent questions about when we should use models o entities.

Reply:
It seems again we have a different naming of things.
I thought that entities were directly linked to database rows; and models more connected to the business/domain.

Reply:
moved to using models